shared/src/commonMain/kotlin/com/boostyboys/mcs/ui/match/MatchDetailsScreen.kt

```diff
@@ -33,45 +37,67 @@ data class MatchDetailsScreen(
     override fun Content() {
         val navigator = LocalNavigator.currentOrThrow
 
-        Surface(
-            modifier = Modifier.fillMaxSize(),
-        ) {
-            Scaffold(
-                topBar = {
-                    McsToolbar(
-                        title = "Match Details",
-                        onBackClicked = {
-                            navigator.pop()
-                        },
-                    )
-                },
-                content = {
-                    Column(
-                        modifier = Modifier.fillMaxSize(),
-                        horizontalAlignment = Alignment.CenterHorizontally,
-                    ) {
-                        Spacer(modifier = Modifier.height(24.dp))
-
-                        Row(
-                            modifier = Modifier
-                                .fillMaxWidth()
-                                .padding(horizontal = 16.dp),
-                            verticalAlignment = Alignment.CenterVertically,
-                            horizontalArrangement = Arrangement.SpaceBetween,
+        AppTheme {
```

Comment:
do we need AppTheme here? shouldn't it be inheriting it from above?

Same comment for the other screens as well